working-groups.md

```diff
@@ -0,0 +1,65 @@
+# Working Groups 
+
+### [Business Planning][biz-wg]
+
+WG for performing feasibility study, and business planning to identify start up costs, fundraising mechanisms, and risks.
```

Comment:
Also sales and grants now!

Reply:
Yes! we should revise

Reply:
Could we more explicitly mention activities that are more generally recognizable as what we've aspired to? Like does any of this sound appropriate:

- writing/updating business plan
- sales and grant application pipelines and support
- communications and outreach (incl social media)
- marketing

Reply:
Ok I take a stab:

```suggestion
WG for strategizing the business goals of the organization, this includes branding and marketing communications, building relationships with clients and neighbours, supporting project teams in sales and grant initiatives, and situating ourselves in the market and solidarity economy toward our organizational mission.
```

Reply:
Sure! Thanks. Thoughts on using bullets for these WG descriptions instead of paragraphs, so verbs and nouns kinda line up, and sub-items are more clear?

(I was planning to lean on bullets when we get to infra's)

Reply:
I prefer sentence over bullet form for something like a high-level description (which is what I took these to be). More detail can be included below. But I think we need a 1 sentence description for each.

Reply:
@benhylau can we discuss this as bizdev? I have many suggestions to streamline the writing and I think it'll be faster synchronously

Reply:
Discussed as a group, this is our proposed revision:
```suggestion
WG for stewarding our business strategy, this includes generating and pursuing business opportunities, relationships, branding, and communications.
```